source/_integrations/ring.markdown

```diff
@@ -23,6 +23,10 @@ ha_platforms:
 ha_integration_type: integration
 ---
 
+<p class='note'>
+This integration currently has a bug that makes it impossible to create automations based on visitors pressing the doorbell. [Issue #88705](https://github.com/home-assistant/core/issues/88705)
+</p>
```

Comment:
We don't document issues, please use and track the issue tracker for that.

Reason is that we have 2K open issue right now, if we would document them all (and have to remove them each time one is solved), it would become unmaintainable.

../Frenck